### PR TITLE
Refactor: Whatsapp config

### PR DESCRIPTION
**Pull request by paulobernardoaf**
Refactor the WhatsApp Integration Config page component.

### CODE REVIEW

File: src/components/config/channels/whatsapp/components/StatusIndicator.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        },
+      },
+    },
+  };
+</script>
+
+<style lang="scss" scoped>
+  .status {
+    height: 8px;
+    border-radius: 50%;
```
Comment (matheusm) [inline src/components/config/channels/whatsapp/components/StatusIndicator.vue:23]: DS Token

File: src/components/config/channels/whatsapp/components/tabs/ConversationsTab.vue [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        margin: $unnnic-spacing-stack-xs 0;
+      }
+    }
+
+    &__buttons {
+      display: flex;
+      justify-content: flex-end;
+
+      &__close {
+        width: 50%;
```
Comment (matheusm) [inline src/components/config/channels/whatsapp/components/tabs/ConversationsTab.vue:164]: ds token

File: src/components/config/channels/whatsapp/components/tabs/GeneralTab.vue [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+            <unnnic-icon-svg
+              class="general-tab__content__info__account__name__icon"
+              icon="single-neutral-2"
+              scheme="neutral-cloudy"
+              size="sm"
+            />
+            <div class="general-tab__content__info__account__name__text">Weni Tecnologia</div>
+          </div>
+          <div class="general-tab__content__info__account__business">
+            Weni Tech - FB 881768118574045
```
Comment (matheusm) [inline src/components/config/channels/whatsapp/components/tabs/GeneralTab.vue:19]: Is it static?

**Review by matheusm — APPROVED** [review]

File: src/components/config/channels/whatsapp/components/tabs/GeneralTab.vue (hunk 3 above)
Comment (paulobernardoaf) [reply src/components/config/channels/whatsapp/components/tabs/GeneralTab.vue:19]: yes for now

**Pull request merged**